private_sharing/models.py

```diff
@@ -315,7 +336,7 @@ def join_url(self):
 
     @property
     def connect_verb(self):
-        return 'join' if self.type == 'on-site' else 'connect'
+        return 'Join' if self.type == 'on-site' else 'Connect'
```

Comment:
was there a reason to change capitalization here?

in templates, text can be passed to "title" to capitalize it, e.g. here https://github.com/OpenHumans/open-humans/blob/master/open_humans/templates/partials/activity-management-join-add-button.html#L8